src/main/kotlin/org/rust/lang/core/psi/RsPsiFactory.kt

```diff
@@ -352,11 +352,13 @@ class RsPsiFactory(
         createFromText<RsFunction>("unsafe fn foo(){}")?.unsafe
             ?: error("Failed to create unsafe element")
 
-    fun createFunction(
-        text: String
-    ): RsFunction =
+    fun createFunction(text: String): RsFunction =
         createFromText(text)
-            ?: error("Failed to create function element: text")
+            ?: error("Failed to create function element: $text")
+
+    fun createRetType(ty: Ty): RsRetType =
```

Comment:
Maybe it's better to make this API more generic and use `String` instead of `Ty`?